api/docs/release.dox

```diff
@@ -238,6 +238,8 @@ Further non-compatibility-affecting changes include:
    setting up the code cache prior to execution.
  - Added support for Windows 10 1803. We provide an artificial version
    identifier #DR_WINDOWS_VERSION_10_1803 to distinguish this major update.
+ - Added module_mapper_t, which factors out of raw2trace_t the module mapping
```

Comment:
`#` for links for both types.

Also, please include a sentence about `#module_mapper_t` replacing raw2trace_t::foo(), etc. which are now deprecated.

Reply:
Re. the "All APIs on this type [..] may fail" - it's true now: find_mapped_trace_address also relays its error to get_last_error(). This way, the type uses one pattern (forced by its constructor), and the comment is true.

BTW, I was thinking - if this "last_error" pattern sticks out, I could do this instead:
- hide the ctor
- replace construction with a "create" static API, which returns a unique_ptr. The "create" returns a string error
- all the other APIs return error strings.

Let me know if that would be preferable.

Reply:
OK, I did not notice that change (just noticed the un-replied-to comment).  This pattern doesn't stick out so much b/c there are similar things in the simulator classes.  It works for me.